pkg/loader/loader.go

```diff
@@ -117,6 +118,9 @@ func (l *Loader) PreLoad(c eval.Context) {
 		// Go plugins
 		l.loadPlugins(c, dir)
 
+		// load typesets from files (NOTE: this is typesets only by convention only at this point, it would attempt to load workflows if found)
```

Comment:
I reckon loading from the "types" dir needs to happen before we load any plugins. Notionally, a Go plugin might someday define a workflow that uses types define in some other language.